docs/FRSKY_R9.md

```diff
@@ -1,8 +1,8 @@
-# mLRS Documentation: Frsky R9M, R9MX #
+# mLRS Documentation: Frsky R9M, R9M Lite Pro, R9MX #
 
 ([back to main page](../README.md))
 
-The Frsky R9M transmitter module and R9MM and R9MX receivers are commercially available and hence can be a relatively easy way to get started with mLRS. However, the R9 hardware does have some limitations as detailed below which may affect some applications.
+The Frsky R9M and R9M Lite Pro transmitter modules and R9MX and R9MM receivers are commercially available and hence can be a relatively easy way to get started with mLRS. However, the R9 hardware does have some limitations as detailed below which may affect some applications.
```

Comment:
Perhaps:

'The Frsky R9M, R9M Lite Pro transmitter modules and R9MX, R9MM receivers can be a relatively easy way to get started with mLRS as they are commercially available.'